index.md

```diff
@@ -8,18 +8,19 @@ with a Domain Driven approach.
 ## The Project
 
 ### Description
-At the moment, waste is collected by garbage trucks regularly from collection points distributed across the cities' streets.
-At each collection point there are several dumpsters for each waste type. 
-People can go to collection points, open a dumpster manually and dispose of the waste.
-There is also extraordinary waste that is not collected at collection points.
-Instead, it must be brought directly to disposal points by citizens.
-
-We aim to optimize the waste collection process using smart technologies.
-
-The idea is to collect waste from collection points only when it is necessary.
-A single garbage truck collects a specific type of waste from various collection points.
-Dumpsters are provided with sensors in order to identify when they must be emptied.
-This sensors provide information about the state of the dumpster itself, not only about its remaining capacity.
+The aim of the project is to optimize the management of waste collection using smart technologies. 
+
+At the moment, waste is collected by **garbage trucks** regularly from **collection points** distributed across the cities' streets.
+At each collection point there are several **dumpsters** for each **waste type**. 
+People can go to collection points, open a dumpster manually and dispose of the **waste**. A single garbage truck collects a specific type of waste from various collection points.
+
+There are also **extraordinary waste** types that are not collected at collection points.
+Instead, them must be brought directly to disposal points by citizens.
+
+The optimization we want to make to the infrastructure is to collect waste from collection points only when necessary, avoiding wasted garbage trucks missions.
```

Comment:
In generale la nuova impostazione di questo paragrafo mi piace molto di più e la trovo molto più chiara. Forse l'unica frase che non  mi convince molto è questa, provo a proporre un'alternativa che non convince molto neanche a me, quindi si accettano suggerimenti

```suggestion
With this project the waste collection process will be optimize using smart technologies. In this way, waste will be collect from collection points only when necessary.
```